netsgiro/objects.py

```diff
@@ -104,9 +104,9 @@ def get_assignments(records: List[Record]) -> List[Assignment]:
 @attr.s
 class Transaction(Serializable):
     service_code = attr.ib(convert=netsgiro.ServiceCode)
-    type = attr.ib(convert=netsgiro.AvtaleGiroTransactionType)  # TODO: Type
+    type = attr.ib(convert=netsgiro.TransactionType)
     number = attr.ib()
-    due_date = attr.ib()
+    nets_date = attr.ib()
```

Comment:
I've been thinking in the same direction. The lower level is now quite complete and uses the same classes for the same concept across services/assignments. The higher level is not complete, especially not wrt. OCR Giro. I guess a lot of this will become evident from using the high level objects to create new files.